### DISCUSSION TITLE
is there a way to control release suffix? <version>.alpha instead of <version>a

### DISCUSSION THREAD

**Discussion by heitorlessa** [python-poetry/poetry#9496] [Q&A]
When using `poetry version prerelease`, I'd like to set a different suffix (`alpha`) though I didn't see as an option. At a first glance, it looks like it's not possible, as it's pre-defined in poetry-core: https://github.com/python-poetry/poetry-core/blob/f088ee23c8dce26af52eb2d94b71755c83e1899d/src/poetry/core/version/pep440/segments.py#L11

Tks a lot!

**Comment by Secrus** [comment]
No. Poetry is following the [PEP 440](peps.python.org/pep-0440/) guidelines for versioning schemes and unless the spec changes, we won't.
